airflow/contrib/hooks/ssh_hook.py

```diff
@@ -90,14 +90,10 @@ def __init__(self,
         # Use connection to override defaults
         if self.ssh_conn_id is not None:
             conn = self.get_connection(self.ssh_conn_id)
-            if self.username is None:
-                self.username = conn.login
-            if self.password is None:
-                self.password = conn.password
-            if self.remote_host is None:
-                self.remote_host = conn.host
-            if self.port is None:
-                self.port = conn.port
+            self.username = conn.login
```

Comment:
I think this should not be changed (and BTW the comment above is wrong). It's the username/password/host/port if specified override the connection, not the other way round. And that's intended behaviour.

Reply:
My bad. Thought I saw those values being initialized as None at __init__. Must be from some other revision.

Reply:
My bad. Thought I saw those values being initialized as None at __init__. Must be from some other revision.